CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+#### [v0.9.4](https://github.com/heptio/ark/releases/tag/v0.9.4) - 2018-08-10
```

Comment:
We'll need to update the date -- let's just hold this PR until the day of release

Reply:
Yep, I figured as much